pkg/app/app.go

```diff
@@ -3,15 +3,19 @@ package app
 import (
 	"fmt"
 	"github.com/fusor/ansible-service-broker/pkg/ansibleapp"
+	"github.com/fusor/ansible-service-broker/pkg/broker"
 	"github.com/fusor/ansible-service-broker/pkg/dao"
+	//"github.com/fusor/ansible-service-broker/pkg/handler"
+	//"net/http"
```

Comment:
Yeah, I left these in along with the calls in `App::Start()` to just indicate the intention here and to allow for it to easily be turned on for testing purposes. Long term, both the imports are necessary.